### PR TITLE
New new keychain docs based on new electron APIs

### PR DESCRIPTION
**Pull request by TylerLeonhardt**

### CODE REVIEW

File: docs/editor/settings-sync.md [hunk 1]
```diff
@@ -144,37 +144,58 @@ Settings Sync uses a dedicated service to store settings and coordinate updates.
...
-If you are experiencing issues with encryption not being available, please follow the steps below:
+#### Other Linux desktop environments
 
-1. Check if the environment variable `XDG_CURRENT_DESKTOP` is defined. You can do this by opening a terminal and running the following command:
+First off, if your desktop environment wasn't detected, [please open an issue on VS Code](https://github.com/microsoft/vscode/issues/new/choose) with the verbose logs from above. This is important for us to support additional desktop configurations.
 
-   ```bash
-   echo $XDG_CURRENT_DESKTOP
-   ```
+##### (recommended) Configure the keyring to use with VS Code
```
Comment (rzhao271) [inline docs/editor/settings-sync.md:184]: Is this really only for the KDE section, or does it apply to other desktop environments as well?
I'm also unsure whether the site renders h5's properly.

Comment (TylerLeonhardt) [reply docs/editor/settings-sync.md:184]: It's under the `Other Linux desktop environments` section.

**Review by rzhao271 — APPROVED** [review]

File: docs/editor/settings-sync.md (hunk 1 above)
Comment (TylerLeonhardt) [reply docs/editor/settings-sync.md:184]: If h5's don't work I guess I could just bold them

Comment (rzhao271) [reply docs/editor/settings-sync.md:184]: It is supported.
I'll create an item to bold them on my side.
Ref https://code.visualstudio.com/docs/java/java-refactoring#_convert-to-lambda-expression

**Pull request merged**